src/Appwrite/Extend/Exception.php

```diff
@@ -73,6 +73,8 @@ class Exception extends \Exception
     public const USER_PHONE_ALREADY_EXISTS         = 'user_phone_already_exists';
     public const USER_PHONE_NOT_FOUND              = 'user_phone_not_found';
     public const USER_MISSING_ID                   = 'user_missing_id';
+    public const USER_EMAIL_ALREADY_VERIFIED        = 'user_email_already_verified';
+    public const USER_PHONE_ALREADY_VERIFIED        = 'user_phone_already_verified';
```

Comment:
Values are not aligned with the rest.

```suggestion
    public const USER_EMAIL_ALREADY_VERIFIED       = 'user_email_already_verified';
    public const USER_PHONE_ALREADY_VERIFIED       = 'user_phone_already_verified';
```

Reply:
fixed